modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -46,10 +48,11 @@ spec:
 <3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
 <4> The actual name of the Container Network Interface (CNI) plug-in that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
 <5> The name of the Linux bridge configured on the node.
-<6> Optional: The VLAN tag.
-<7> The CNI plug-in that allows the MAC pool manager to assign a unique MAC address to the connection.
+<6> The MAC spoof check. When set to `true`, this feature provides security against a MAC spoofing attack where an intruder impersonates a device with a valid MAC address to bypass access control lists and breach the network.
```

Comment:
I would prefer to see a description of what it does and not why it should be used. At least, the first part if not clear and the second is to accurate.

This option provides security against a MAC spoofing attack by allowing only a single MAC address (which cannot be changed ) to exit the pod.

(if the user will try to change the MAC, traffic will be blocked)

I think the second part is not accurate because this is not protecting against an external intruder that may attack the pod from outside. So IMO just describing what it does should be enough.

Reply:
Reworded per your suggestion.